nix/tests/lanzaboote.nix

```diff
@@ -267,4 +267,56 @@ in
         f"Expected: {expected_loader_config} is not included in actual config: '{actual_loader_config}'"
     '';
   };
+
+  export-efi-variables = mkSecureBootTest {
+    name = "lanzaboote-exports-efi-variables";
+    testScript = ''
+      import struct
+
+      machine.start()
+      SD_LOADER_GUID = "4a67b082-0a4c-41cf-b6c7-440b29bb8c4f"
+      expected_variables = ["LoaderDevicePartUUID",
```

Comment:
A few of these variables might be/will be set by the boot loader. The stub would only set them if the boot loader didn't. So if we wanted to test this fully we would need to get rid of sd-boot and boot the stub directly (e.g. via a custom bootloader).

Reply:
We should definitely do that in those tests

Reply:
Did a trick to do it